lib/AddDependencies.js

```diff
@@ -38,7 +43,7 @@ class AddDependencies {
             break;
           }
           default: {
-            if (/package\.json/.test(val)) {
```

Comment:
Shouldn't these kind of changes should be split to separate PRs?

Reply:
yeah i should have, after refactoring this pr out into separate ones i found it wasn't needed.